src/components/Analyze/AnalyzeChoice.tsx

```diff
@@ -1,42 +1,49 @@
 import { AnalyzeContext } from '@/components/Analyze/AnalyzeContext'
+import { DatePick } from '@/utils/DatePick'
 import { addComma } from '@/utils/money'
 import { useContext } from 'react'
 
 export const AnalyzeChoice = () => {
-	const { totalPrice } = useContext(AnalyzeContext)
+	const { totalPrice, startDate, endDate, setEndDate, setStartDate } =
+		useContext(AnalyzeContext)
+
+	console.log('choice')
+
 	return (
 		<div className='flex flex-initial flex-col tablet:flex-row gap-1 justify-start w-full tablet:gap-10 px-10'>
 			<div className='relative flex-1 flex flex-col justify-start items-center bg-[#f6f6f7] rounded-[10px] shadow-analyze-box w-full h-full p-[20px]'>
-				<div className='flex flex-col justify-between items-center analyzechoice:flex-row flex-1 gap-2 w-full'>
-					<div className='flex flex-1 flex-col'>
+				<div className='flex flex-row justify-evenly items-center flex-1 gap-2 w-full'>
+					<div className='flex flex-initial flex-col'>
```

Comment:
![image](https://github.com/user-attachments/assets/810d9742-e742-43cd-b1c8-43ca43641004)

```suggestion
				<div className='flex flex-row justify-between items-center flex-1 gap-2 w-full'>
					<div className='flex flex-1 flex-col'>
```
`DatePick` 모두 `flex-1`으로 설정하면 될 것 같습니다! 그럼 자식요소들이 부모 요소를 flex하게 차지합니다.

테스트해보니 태블릿 너비 이하에서는 가운데 정렬이 되어버려 보기 안 좋은 상황이 일어나네요. 브레이크 포인트를 추가해주어 태블릿 너비 이하에서는 왼쪽 정렬이 되게 만들면 될 것 같습니다.
```suggestion
				<div className='flex flex-row justify-between items-start tablet:items-center flex-1 gap-2 w-full'>
					<div className='flex flex-1flex-col'>
```

요소들의 크기를 존중하고 싶다면 initial을 써도 무방하지만 현재 데스크탑에서 너무 가운데 정렬이 되어 보이니 `flex-1`이 좀 더 괜찮은 방식 같습니다.
`evenly`와 `between`은 취향? 차이라고 생각하니 원하시는 것으로 사용하시면 될 듯합니다. 어차피 요소가 두개라서 양 끝에 붙든 모든 공간을 동일하게 분배하든 `flex-1`로 자식요소들이 공간을 차지하면 같은 결과가 나오더라구요.

Reply:
여러가지 시도끝에 이쁘게 맞춰봤는데 한번 평가좀 해주십쇼
![2025-02-1411 51 47-ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/9f47369a-cc49-4afc-bf8c-7ca402cba350)